server/game/gameSystems/SelectCardSystem.ts

```diff
@@ -32,16 +31,15 @@ export interface ISelectCardProperties extends ICardTargetSystemProperties {
 
 /**
  * A wrapper system for adding a target selection prompt around the execution the wrapped system.
- * Only used for adding a selection effect to a system that is part of a cost.
+ * Functions the same as a targetResolver and used in situations where one can't be created (e.g., costs).
  */
-// TODO: why is this class needed for evaluating costs when systems already have target evaluation and selection built in?
 export class SelectCardSystem extends CardTargetSystem {
```

Comment:
I think we should add to the Readme something that says what each system is used for. It's not clear to me what situations call for SelectCard vs. not

Reply:
Agreed. There are some missing pieces that will come in the Ambush PR, at that point I'll have enough to fill out a readme section on how this is all used.